index.bs

```diff
@@ -1212,20 +1212,51 @@ dictionary MLClampOptions {
 };
 
 partial interface MLGraphBuilder {
-  MLOperand clamp(MLOperand x, optional MLClampOptions options = {});
+  MLOperand clamp(MLOperand operand, optional MLClampOptions options = {});
   MLActivation clamp(optional MLClampOptions options = {});
 };
 </script>
 <div algorithm=clamp>
     **Arguments:**
-        - *x*: an {{MLOperand}}. The input tensor.
+        - *input*: an {{MLOperand}}. The input tensor.
         - *options*: an optional {{MLClampOptions}}. The optional parameters of the operation.
             - *minValue*: a {{float}} scalar. Specifies the minimum value of the range. When it is not specified, the clamping is not performed on the lower limit of the range.
             - *maxValue*: a {{float}} scalar. Specifies the maximum value of the range. When it is not specified, the clamping is not performed on the upper limit of the range.
 
     **Returns:**
-        - an {{MLOperand}}. The output tensor of the same shape as *x*.
-        - an {{MLActivation}}. The activation function representing the clamp operation.
+        - an {{MLOperand}}. The output tensor of the same shape as *input*.
+        - an {{MLActivation}}. The operator representing the clamp operation.
+
+    The {{MLGraphBuilder/clamp()}} method steps are:
+    1. Let |arg1| be the first argument.
+    1. If |arg1| is an instance of {{MLOperand}}, run the {{MLGraphBuilder/clamp(operand, options)}} steps with all the received arguments.
+    1. Otherwise, run the {{MLGraphBuilder/clamp(options)}} steps with all received arguments.
```

Comment:
@huningxin : in general, for polymorphic functions, do we prefer adding these boilerplate lines, or just define the two forms separately?